src/services/company.js

```diff
@@ -0,0 +1,18 @@
+class CompanyService {
+    // TODO
+    constructor() {
+
+    }
+
+    getOffersInTimePeriod(owner, publishDate, publishEndDate, offers) {
+        return offers.find({
+            owner,
+
+            $or: [{ $and: [{ publishEndDate: { $gte: publishDate } }, { publishEndDate: { $lte: publishEndDate } }] },
+                { $and: [{ publishDate: { $gte: publishDate } }, { publishDate: { $lte: publishEndDate } }] },
+                { $and: [{ publishDate: { $lte: publishDate } }, { publishEndDate: { $gte: publishEndDate } }] }]
+        });
+    }
```

Comment:
```suggestion
            $or: 
                [{ $and: [{ publishEndDate: { $gte: publishDate } }, { publishEndDate: { $lte: publishEndDate } }] },
                { $and: [{ publishDate: { $gte: publishDate } }, { publishDate: { $lte: publishEndDate } }] },
                { $and: [{ publishDate: { $lte: publishDate } }, { publishEndDate: { $gte: publishEndDate } }] }
            ]
        });
    }
```

Improve code readability